src/extension.ts

```diff
@@ -93,9 +93,13 @@ async function startServer(tomcat: TomcatController, tomcatItem ?: TomcatServer)
     }
 }
 
-async function stopServer(tomcat: TomcatController, tomcatItem ?: TomcatServer): Promise<void> {
+async function stopServer(tomcat: TomcatController, tomcatItem?: TomcatServer): Promise<void> {
     const server: TomcatServer = await getTargetServer(tomcat, tomcatItem);
     if (server) {
+        if (!server.isStarted()) {
+            vscode.window.showInformationMessage('This Tomcat Server was stopped.');
```

Comment:
Should this message also in the Dialogmessage constants?

Reply:
yes, but that PR is in review, so just use pure string here.